docs/apis/calcite.md

```diff
@@ -0,0 +1,529 @@
+Apache Apex,unified stream processing engine enables application developers to process data at very high throughput with low latency. Though type of data demands different type of
+processing requirements but SQL remains way to process input data .To ensure that existing ETL developers,developers who are well versed with Database applications adopt stream processing appliction development with ease, integration of SQL  with Apex was needed. Being popular and Apache project, Calcite was chosen for an integration and to facilitate use of SQL with Apex.
```

Comment:
Being a popular Apache project, Apache Calcite was chosen for an integration and to facilitate use of SQL with Apex